### PR TITLE
[16.0][IMP] connector_search_engine: check binding model

### PR DESCRIPTION
**Pull request by qgroulard**

### CODE REVIEW

**Review by flotho — APPROVED** [review]
Code review.
Not so much changes,
LGTM

File: connector_search_engine/tests/test_all.py [hunk 1]
```diff
@@ -381,3 +383,9 @@ def test_binding_multi_backend_index(self):
         self.assertEqual(self.partner_binding.mapped("state"), ["done"])
         bindings.delete_record()
         self.assertFalse(self.partner_binding.exists())
+
+    def test_binding_wrong_model(self):
+        # Try to add a 'res.users' record to a 'res.partner' index
+        user = self.env["res.users"].search([], limit=1)
+        with self.assertRaises(ValidationError):
```
Comment (simahawk) [inline connector_search_engine/tests/test_all.py]: can you test the error msg w/ `assertRaisesRegex` pls?

**Review by simahawk — COMMENTED** [review]
LG, small remark on the tests

**Comment by lmignon** [comment]
@qgroulard Can you take into account the last comment from Simone?

**Pull request merged**